api/app/utils/redapp.py

```diff
@@ -0,0 +1,316 @@
+    public double conifMixedWood;
+    public double deadBalsam;
+    public double grassCuring;
+    public double grassFuelLoad;
+    public double crownBase;
+    public double elapsedTime;
+    public double ffmc;
+    public double windSpeed;
+    public double windDirection;
+    public double bui;
+    public double dmc;
+    public double dc;
+    public double slopeValue;
+    public double aspect;
+    public double latitude;
+    public double longitude;
+    public double elevation;
+    public boolean acceleration;
+    public boolean useCrownBaseHeight;
+    public boolean useBui;
+    public boolean useBuildup;
+    public boolean useSlope;
+    public boolean useGreenup;
+    public int fuelType;
+    public String fireDescription;
+    public String headFireDescription;
+    public String flankFireDescription;
+    public String backFireDescription;
+
+    public Calendar m_date;
+
+    public boolean cfbPossible = true;
+    """
+
+    def __init__(self):  # pylint: disable=too-many-statements
+        """ Init variables """
+        # pylint: disable=invalid-name
+        self.ros_t: float = None
+        self.ros_eq: float = None
+        self.fros: float = None
+        self.bros: float = None
+        self.rso: float = None
+        self.hfi: float = None
+        self.ffi: float = None
+        self.bfi: float = None
+        self.area: float = None
+        self.perimeter: float = None
+        self.distanceHead: float = None
+        self.distanceFlank: float = None
+        self.distanceBack: float = None
+        self.lb: float = None
+        self.csi: float = None
+        self.cfb: float = None
+        self.sfc: float = None
+        self.tfc: float = None
+        self.cfc: float = None
+        self.isi: float = None
+        self.fmc: float = None
+        self.wsv: float = None
+        self.raz: float = None
+        self.conifMixedWood: float = None
+        self.deadBalsam: float = None
+        self.grassCuring: float = None
+        self.grassFuelLoad: float = None
+        self.crownBase: float = None
+        self.elapsedTime: float = None
+        self.ffmc: float = None
+        self.windSpeed: float = None
+        self.windDirection: float = None
+        self.bui: float = None
+        self.dmc: float = None
+        self.dc: float = None
+        self.slopeValue: float = None
+        self.aspect: float = None
+        self.latitude: float = None
+        self.longitude: float = None
+        self.elevation: float = None
+        self.acceleration: bool = None
+        self.useCrownBaseHeight: bool = None
+        self.useBui: bool = None
+        self.useBuildup: bool = None
+        self.useSlope: bool = None
+        self.useGreenup: bool = None
+        self.fuelType: int = None
+        self.fireDescription: str = None
+        self.headFireDescription: str = None
+        self.flankFireDescription: str = None
+        self.backFireDescription: str = None
+        self.m_date: date = None
```

Comment:
what's `m_date` for?

Reply:
Presumably the date used to calculate the date of minimum foliar content?